tools/wakebox/wakebox.cpp

```diff
@@ -72,6 +72,14 @@ void print_help() {
       "  \n"
       "    -i --allow-interactive   Use default stdin, ignoring the params json file's stdin "
       "value.  \n"
+      "    -b --bind DIR1:DIR2      Place the directory (or file) at DIR1 within the command's "
+      "view  \n"
+      "                             of the filesystem at location DIR2.                            "
+      "  \n"
+      "                             May be specified multiple times.                               "
+      "  \n"
+      "                             These are are applied after the params file mount ops.         "
+      "  \n"
```

Comment:
Does this get printed on MacOS?

Reply:
I haven't built this branch on a Mac yet, but I believe it would. I can add an ifdef around this section, though maybe now's the time to mention that the strings here are really difficult to read because it looks like the string literals have been wrapped at a short line length than the strings they contain, causing the newline characters and sometimes even the last word of the printed line to appear on the following source line.

Should I fix up the string literals in this file first?

Reply:
Yeah an #ifdef is ok
I'm guessing the formatting was overlooked when the clang-format was applied to the whole repo

Reply:
This new text is now guarded by an `#ifdef`.